Kickstarter-iOS/Views/Controllers/RewardPledgeNavigationController.swift

```diff
@@ -32,8 +32,6 @@ extension RewardPledgeNavigationController: UINavigationControllerDelegate {
     case (.pop, is PledgeViewController, is RewardPledgeTransitionAnimatorDelegate):
       return RewardPledgePopTransitionAnimator()
     default:
-      // We're not performing any custom transition, bring back the standard divider line
-      self.navigationBar.shadowImage = nil
```

Comment:
Removed this line in favor of keeping the shadow for all VCs managed by `RewardPledgeNavigationController`. This will cause some funkiness with colors in the deprecated pledge VC and possibly the Thanks page, but I think we can easily fix those during a design pass - ideally we can get to a place where there's visual consistency between all the VCs in the checkout flow.

Reply:
Can you describe what funkiness do you have in mind? 😄

Reply:
I believe what might happen is the line won't come back in the `Thanks` VC after checkout. What I think we should do is do a light colors pass on the `Thanks` screen to make it visually consistent with the other screens in the new checkout flow.